relay-server/src/actors/events.rs

```diff
@@ -1176,7 +1156,10 @@ impl EventProcessor {
                 self.expand_unreal(&mut state)?;
             });
 
-            self.extract_event(&mut state)?;
+            self.extract_event(&mut state).map_err(|error| {
+                log::error!("failed to extract event: {}", LogError(&error));
+                error
+            })?;
```

Comment:
Why did we have to `log::error!` inside a `with_scope` in the old `extract_event` and not here ?

Reply:
I missed that we already run this entire function in a `with_scope`, so it was redundant.